pkg/api/pod/warnings.go

```diff
@@ -237,15 +237,15 @@ func warningsForPodSpecAndMeta(fieldPath *field.Path, podSpec *api.PodSpec, meta
 	// use of pod seccomp annotation without accompanying field
 	if podSpec.SecurityContext == nil || podSpec.SecurityContext.SeccompProfile == nil {
```

Comment:
> I think we should always return the warning when the annotation is set, otherwise users will never see it since we copy the annotation to the field. Same applies below (per-container warning).

Makes sense, I'll remove the checks for `podSpec.SecurityContext == nil || podSpec.SecurityContext.SeccompProfile == nil`

> Also, is this called on PodTemplates embedded in workload resources? I think it would be good to return a warning in those cases too, as that's the most common way to create pods.

Yes, see: https://github.com/kubernetes/kubernetes/blob/2a017f94bcf8d04cbbbbdc6695bcf74273d630ed/pkg/api/pod/warnings.go#L60